### DISCUSSION TITLE
Bulbs don't work after upgrade from .2

### DISCUSSION THREAD

**Discussion by kasper79** [sbidy/wiz_light#105] [Q&A]
Hello!

The integration has been working great for awhile! I recently upgraded to .3 and .3.1 and only one of my bulbs continue to work. I removed all of them and re-added them via UI and again via .yaml with no luck. They continue to work via WIZ app and I can see them on my network (static IP's). If I roll back to .2, then add them into my config yaml they all work again in hass.

Model(s) that doesn't work with .3.x:
ESP05_SHRGBL_21 (5 of these)
ESP01_SHRGB_23 (2 of these)

Model that DOES work with .3.x:
ESP01_SHRGB1C_31

Please let me know what info I can provide or anything to try and test on my end. Running core-2021.2.1.

Thank you!

**Comment by sbidy** [comment]
Yes, there are Issues with the 0.3.1. From 0.2 to 0.3 are some major changes made.
Please check the (one) related issue #99 for more details.

If I release the new version it would be great if you can test because there are so many different bulb types out there.
I have only 2 of them and I don't want to buy "all/multiple" bulbs only for testing. For that they are to expensive.

Maybe I get WiZ as sponsor for the project 😄 🚀

**Reply by kasper79** [reply]
Thanks for the reply! AND thank you for all the work being done to give us Wiz support in HASS!!!!!

I will gladly test any release as I have about 4-5 different "Wiz" bulbs now.

I saw that thread ( #99 ) and unfortunately haven't got mine to work yet. So I will monitor and look for any beta/GA releases to try and test.

Thanks again!!

Cheers

**Reply by atkinsonar** [reply]
@sbidy  - Appreciate the dedication to getting this working.  I'll echo the above, but am in an opposing situation with the latest release v0.3.1...

All bulbs are static IPs and work from the app.  I have 11 other bulbs that are working great from HA (mix of the A19 color/tunable white and the recessed downlight kits).

I have x1 WiZ Quest Projector that is not currently being recognized.  Let me know if I can help with any testing on this.

https://www.homedepot.com/p/WiZ-Quest-6-38-in-White-Colors-and-Tunable-White-Wi-Fi-Connected-Smart-LED-Projector-Lamp-IZ730109/310792591
